src/init.js

```diff
@@ -54,10 +55,20 @@ define([
 		function translate(load){
 			var result = parse(load.source, this, zoneOpts);
 
+			// Register dynamic imports for the slim loader config
+			var localLoader = loader.localLoader || loader;
+			if (localLoader.slimConfig) {
+				var toMap = localLoader.slimConfig.toMap;
+				var res = getIntermediateAndImports(load.source);
```

Comment:
This is already part of the `result` object, this is done by the `./parse` module.